d3a_interface/sim_results/__init__.py

```diff
@@ -50,6 +50,17 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def is_house_node_type(area):
+    return get_unified_area_type(area) == "Area"
+
+
+def has_no_grand_children(area):
+    for child in area.get("children", []):
+        if child.get("children", []):
+            return False
+    return True
```

Comment:
Nit: this negative method is a bit "twisted". Maybe it would be better to have `has_grandchildren` and use it like: `if not has_grandchildren(area)`. What do you think?

Reply:
To me it looks getting twisted after explicitly adding `not` in `has_grandchildren `, nevertheless I'm indifferent and will adapt as you like here.